### ISSUE TITLE
"make homepage" route in routing auto tutorial is insecure

### ISSUE THREAD

**Issue by dantleech** [symfony-cmf/symfony-cmf-docs#680]
http://symfony.com/doc/current/cmf/tutorial/make-homepage.html

as on IRC:

````
 | A simple fix would be to change the root to                                    
 | - /admin/action/make_homepage/                                                 
 | then when firewalls are configured for the admin bundle they apply to that     
 | route also                                                                     
````

**Comment by dbu** [comment]
and also makeHomepageAction should only allow POST and not GET - if i see it correctly it does not specify the method currently.

**Comment by dbu** [comment]
@dantleech did #763 fix this?

**Comment by dantleech** [comment]
No, but https://github.com/symfony-cmf/symfony-cmf-docs/pull/777
would

**Comment by dbu** [comment]
fixed in #777
